README.md

```diff
@@ -44,4 +44,59 @@ pip install zep-python
 
 ```bash
 poetry add zep-python
-```
\ No newline at end of file
+```
+
+## Zep Cloud Installation
+In order to install Zep Python SDK with Zep Cloud support, you will need to install
+a release candidate version.
+
+```bash
+pip install --pre zep-python
+```
+
+-- OR --
+
+```bash
+poetry add zep-python@^2.0.0-rc
+```
+
+You will also need to provide a Zep Project API key to your zep client for cloud support.
+You can find out about Zep Projects in our [cloud docs](https://help.getzep.com/projects.html)
+
+### Using LangChain Zep Classes with `zep-python`
+
+(Currently only available on release candidate versions)
+
+In the pre-release version `zep-python` sdk comes with `ZepChatMessageHistory` and `ZepVectorStore`
+classes that are compatible with [Langchain's Python expression language](https://python.langchain.com/docs/expression_language/)
```

Comment:
@ellipsis-dev please ensure that "LangChain" is spelled correctly in the comments in this PR. Exclude code from your review.